cogs/ahk/help.py

```diff
@@ -495,6 +495,23 @@ async def strip_postfix(self, channel: discord.TextChannel):
 	def is_claimed(self, channel_id):
 		return channel_id in self.claimed_channel.values()
 
+	@commands.command(hidden=True)
+    	async def wta(self,ctx):
+		'''Responds with the currently open for claiming channels.'''
+        	open_category = self.bot.get_channel(OPEN_CATEGORY_ID)
+        	channels = list(open_category.text_channels)
+        	channel_mentions = []
+        	for c in channels:
+        	    	if c.id == GET_HELP_CHAN_ID:
+                		continue
+            	channel_mentions.append(c.mention)
+        	wta_format = ""
+        	if len(channel_mentions) == 0:
+            		wta_format = "No help channels are available for questions at this time. Please wait to ask a question, or ask staff for guidance."
+        	else:
+            		wta_format = "Please ask in a channel under the **{}** category. Use " + (("{}, " *  (len(channel_mentions)-1) + "or ") if len(channel_mentions)-1 > 0 else "") + "{}."
+        	await ctx.send(wta_format.format(open_category.name,*channel_mentions))
```

Comment:
Unpacking channel_mentions here will mean it's referenced by its index in the format string, which isn't how you're accessing it, so fairly certain this format won't work

Reply:
This does work in my tests, although if you have a better method I'm interested in trying it.